tests/gui/environment.py

```diff
@@ -28,6 +29,8 @@ def before_scenario(context, scenario):
     context.driver = webdriver.Chrome(
         service=chrome_service, options=chrome_options)
 
+    context.page_factory = PageObjectFactory()
```

Comment:
You can pass the driver to the PO factory right here, something like this:
PageObjectFactory(context.driver)